src/MartinGeorgiev/ValueObject/Point.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\ValueObject;
```

Comment:
I think a better namespace will be `MartinGeorgiev\Doctrine\DBAL\Types\ValueObject`. This way it will be implied explicitly that this `Point` class is intended to be used in the context of Doctrine types and is not designed to be a generic `Point` implementation.